src/util/migration-spec.test.ts

```diff
@@ -96,7 +96,7 @@ describe('validateSpec', () => {
   });
 
   it('rejects a spec with a missing adapter type', () => {
-    const spec = cloneDeep(baseSpec);
+    const spec: any = cloneDeep(baseSpec);
```

Comment:
This was needed as I was getting the following error running `tsc`:
```bash
src/util/migration-spec.test.ts:100:12 - error TS2790: The operand of a 'delete' operator must be optional.

100     delete spec.adapter.type;
               ~~~~~~~~~~~~~~~~~
```
Basically it's saying that `spec.adapter.type` was declared as required instead of optional, so it can't be deleted. We of course _can't_ make it optional, as that is the point of the test, hence this.